onboarding/password-management.md

```diff
@@ -0,0 +1,59 @@
+# Password management
+
+By working in the laboratory you will be close to very sensitive information such as secure access keys and passwords, for personal or group use, allowing you to gain access to servers or critical physical spaces.
+
+In this context, it is everyone's duty to ensure the security of these sensitive information throughout its use and you should take this responsibility in account in all your actions and decisions.
+
+For example:
+
+ - Taking a picture inside the lab can lead to sensible information leaking on social's network due to some computer screen, note or post-it in the background of the picture
+ - Committing files on git without reread can lead to sensible information leaking on the cloud due to config file containing password or keys 
+
+Although we cannot have total security, simple practices and awareness of laboratory members can already avoid a lot of problems.
+
+## Storage of the sensitive information
+
+Polytechnique Montreal does not provide a secure password system within the university, this responsibility is therefore distributed to each member of the laboratory.
+
+**You should:**
+
+ - Use a protected password manager like `KeyPass`, `Apple's keychain`, `Google password` or `Dashlane`.
+
```

Comment:
Somewhere in here we should add

```suggestion
**You should**:

- Use a unique password for each account. A good password manager will help you do this, and you shouldn't even have to memorize each password.
```

because one of the most common ways data breaches happen is via password reuse. And I know multiple people in the lab reuse their passwords.